app/routes/public/index.js

```diff
@@ -1,12 +1,30 @@
 import Ember from 'ember';
+import moment from 'moment';
 
 const { Route, RSVP } = Ember;
 
 export default Route.extend({
   model() {
     return RSVP.hash({
-      event: this._super(...arguments),
-
+      event   : this._super(...arguments),
```

Comment:
You're calling the super method twice. You should only call once. Store the result in a variable and use it twice.